npm-packages/meteor-installer/package-lock.json

```diff
@@ -0,0 +1,761 @@
+{
+  "name": "meteor",
+  "version": "2.8.0",
```

Comment:
is this version correct?

Reply:
I should not have committed this file. For some reason, there is not a json.lock file in this package